### ISSUE TITLE
download_udpos error

### ISSUE THREAD

**Issue by Liangtaiwan** [google-research/xtreme#23]
I got the following  when download_updos
```
*** AttributeError: 'DependencyTree' object has no attribute 'node'
```
The error occurs in https://github.com/google-research/xtreme/blob/master/third_party/ud-conversion-tools/lib/conll.py.

It seems that in line 380-381, sent only call add_edge but not call add_node.
As a result, there is no node in the graph (Dependency Tree)
https://github.com/google-research/xtreme/blob/df7457ceef611b2ac39600ac5fdf1e9190b30c20/third_party/ud-conversion-tools/lib/conll.py#L350-L390

**Comment by stefan-it** [comment]
Hi @Liangtaiwan ,

I think I had the same issue and solved by installing `networkx` in version 1.11 (so `pip install networkx==1.11` should help) :)

**Comment by Liangtaiwan** [comment]
@stefan-it  Thanks for your help!
